$NetBSD: patch-Makefile,v 1.9 2015/09/16 16:17:18 wiz Exp $

* Add "all" target.
* Use ${PREFIX} variable.
* Install man pages in ${PREFIX}/${PKGMANDIR}, not ${PREFIX}/share/man.

--- Makefile.orig	2015-08-30 20:07:13.000000000 +0000
+++ Makefile
@@ -17,11 +17,12 @@
 VERSION=1.33
 
 .PATH: $(.PARSEDIR)
-prefix?=/usr/local
+prefix?=$(PREFIX)
 target=$(DESTDIR)$(prefix)
 parsedir:=$(.PARSEDIR)
 srcdir=$(dir $(abspath $(firstword $(MAKEFILE_LIST))))$(parsedir)
 VPATH=$(srcdir)
+mandir=$(PKGMANDIR)
 
 INSTALL = install
 
@@ -39,7 +40,7 @@ CFLAGS += -pthread
 CPPFLAGS += -DTHREADS
 
 # Optimizing for speed. Comment this out for distribution builds
-CFLAGS += -march=native
+#CFLAGS += -march=native
 
 # To enable debugging of the Yacc grammar, uncomment the following line
 #CPPFLAGS += -DYYDEBUG=1
@@ -97,6 +98,8 @@ dump.o export.o graph.o main.o merge.o r
 
 BISON ?= bison
 
+all: cvs-fast-export cvs-fast-export.1
+
 gram.h gram.c: gram.y
 	$(BISON)  $(YFLAGS) --defines=gram.h --output-file=gram.c $(srcdir)/gram.y
 lex.h lex.c: lex.l
@@ -137,10 +140,10 @@ install-bin: cvs-fast-export cvssync cvs
 	$(INSTALL) -d "$(target)/bin"
 	$(INSTALL) $^ "$(target)/bin"
 install-man: man
-	$(INSTALL) -d "$(target)/share/man/man1"
-	$(INSTALL) -m 644 cvs-fast-export.1 "$(target)/share/man/man1"
-	$(INSTALL) -m 644 cvssync.1 "$(target)/share/man/man1"
-	$(INSTALL) -m 644 cvsconvert.1 "$(target)/share/man/man1"
+	$(INSTALL) -d "$(target)/${PKGMANDIR}/man1"
+	$(INSTALL) -m 644 cvs-fast-export.1 "$(target)/${PKGMANDIR}/man1"
+	$(INSTALL) -m 644 cvssync.1 "$(target)/${PKGMANDIR}/man1"
+	$(INSTALL) -m 644 cvsconvert.1 "$(target)/${PKGMANDIR}/man1"
 
 PROFILE_REPO = ~/software/groff-conversion/groff-mirror/groff
 gmon.out: cvs-fast-export
